src/common/transformations/src/transformations/common_optimizations/transpose_sinking.cpp

```diff
@@ -61,6 +62,54 @@ std::shared_ptr<ngraph::opset6::Constant> get_reversed_order_constant(const std:
 
 } // namespace
 
+ngraph::pass::TransposeEltwise::TransposeEltwise() {
+    MATCHER_SCOPE(TransposeEltwise);
+
+    auto eltwise_data_input_p = pattern::any_input();
+    auto eltwise_const_input_p = pattern::wrap_type<opset6::Constant>();
+    auto eltwise_p = pattern::wrap_type<op::util::BinaryElementwiseArithmetic>({eltwise_data_input_p, eltwise_const_input_p},
+        [](const Output<Node> & output) {
+            return ov::is_preprocesing_node(output.get_node_shared_ptr());
+    });
+    auto transpose_p = pattern::wrap_type<opset6::Transpose>({eltwise_p,
+                                                              pattern::wrap_type<opset6::Constant>()},
+                                                              pattern::consumers_count(1));
+
+    auto callback = [=](ngraph::pattern::Matcher &m) {
+        const auto &pattern_to_output = m.get_pattern_value_map();
+        auto eltwise = pattern_to_output.at(eltwise_p).get_node_shared_ptr();
+        auto eltwise_const_input = pattern_to_output.at(eltwise_const_input_p);
+        auto eltwise_data_input = pattern_to_output.at(eltwise_data_input_p);
+        auto transpose = pattern_to_output.at(transpose_p).get_node_shared_ptr();
+
+        const auto & order_size = transpose->get_input_shape(1).at(0);
+        const auto & shape = eltwise_const_input.get_shape();
+        if (shape.size() != order_size && ov::shape_size(shape) != 1) {
+            // TODO: temporary restrictions
+            return false;
```

Comment:
basically we can make an unsqueeze like we pull Reshape through FQ

Reply:
Yes, the implementation is limited but it is enough to move preprocessing eltwises through transpose.
Also I was thinking that this sinking will be later replaced by TF FE transpose sinking which supports BinaryElementwiseArithmetic operations.